### DISCUSSION TITLE
Rclone exclude directories

### DISCUSSION THREAD

**Discussion by ToasterSticks** [coder/code-server#3090] [Q&A] [answered]
For any of you using rclone on Railway, what would I pass into the `RCLONE_FLAGS` environment variable for it to exclude all instances of a particular folder name? eg `node_modules`

**Comment by ghost** [comment, marked as answer]
I am currently using `--exclude "node_modules/**" --exclude ".git/**"` but I have also seen `**/node_modules` but the one im using works. Note: I only exclude git because I publish the repos to github.

**Reply by ToasterSticks** [reply]
Haha thanks, I couldn't figure out how the flags worked. It seemed there were too many files or something,so --exclude node_modules/ worked perfectly. Thanks!

**Reply by bpmct** [reply]
I'll be sure to document this!

issue: https://github.com/cdr/deploy-code-server/issues/18

**Reply by bpmct** [reply]
this has been documented, thanks!
